docs/sources/flow/setup/start-agent.md

```diff
@@ -0,0 +1,123 @@
+---
+description: Learn how to start, restart, and stop Grafana Agent Flow after it is installed
+title: Start, restart, and stop Grafana Agent Flow 
+menuTitle: Start
+weight: 800
+---
+
+# Start Grafana Agent Flow
```

Comment:
Kubernetes and Docker are our biggest installation targets, but there's no information for them here, which might make those users feel like there's missing documentation for their platform.

Should we include some kind of Kubernetes and Docker instructions?

Reply:
Yes we need Kubernetes and Docker info. This is missing in the current docs and became very obvious when I split the topics up by task